recipes/nasm/all/conanfile.py

```diff
@@ -103,15 +95,18 @@ def package(self):
         copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
         if is_msvc(self):
             copy(self, pattern="*.exe", src=self.source_folder, dst=os.path.join(self.package_folder, "bin"), keep_path=False)
+            with chdir(self, os.path.join(self.package_folder, "bin")):
+                shutil.copy2("nasm.exe", "nasmw.exe")
+                shutil.copy2("ndisasm.exe", "ndisasmw.exe")
         else:
             autotools = Autotools(self)
             autotools.install()
             rmdir(self, os.path.join(self.package_folder, "share"))
 
     def package_info(self):
-        bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH environment variable: {}".format(bin_path))
-        self.env_info.PATH.append(bin_path)
-        self.env_info.libdirs = []
-        self.buildenv_info.append_path("PATH", bin_path)
+        if Version(conan_version).major < 2:
+            bin_path = os.path.join(self.package_folder, "bin")
+            self.output.info("Appending PATH environment variable: {}".format(bin_path))
+            self.env_info.PATH.append(bin_path)
```

Comment:
We've decided that we dont want to add a version check, since this is only for old generators plus its not used by other recipes https://conan.io/center/nasm?tab=usedby

it can be dropped :)

```suggestion
```